### PR TITLE
[11.0][FIX] stock_request: added readonly true on related company fields

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This solver an issue when a non Administraator rights try to create a stock request order related company fields try to be mnodified promping next error:
image

### CODE REVIEW

File: stock_request/models/stock_request_abstract.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def _compute_product_qty(self):
     )
     allow_virtual_location = fields.Boolean(
         related='company_id.stock_request_allow_virtual_loc',
+        readonly=1,
```
Comment (rousseldenis) [inline stock_request/models/stock_request_abstract.py]: True

File: stock_request/models/stock_request_order.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def _get_default_requested_by(self):
     )
     allow_virtual_location = fields.Boolean(
         related='company_id.stock_request_allow_virtual_loc',
+        readonly=1,
```
Comment (rousseldenis) [inline stock_request/models/stock_request_order.py]: True

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. This behaviour has been changed in Odoo v12. related fields are now readonly by default.

**Comment by HviorForgeFlow** [comment]
Good to know @rousseldenis :smile:
Move it forward
Thanks

**Review by rousseldenis — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@etobella @lreficent could you review this?

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks

**Pull request merged**